modules/database/src/admin/schema.admin.ts

```diff
@@ -38,45 +38,57 @@ export class SchemaAdmin {
             updatedAt: 1,
           },
         },
-      )
-      .then(r => {
-        return { schemas: r };
-      });
+      );
+    const cndSchemas = await this.database
+      .getSchemaModel('_DeclaredSchema')
+      .model.findMany(
+        { 'modelOptions.conduit.cms': { $exists: false } },
+        {
+          select: 'name extensions',
+          sort: {
+            updatedAt: 1,
+          },
+        },
+      );
+    return { schemas: cmsSchemas.concat(cndSchemas) };
   }
 
-  async importCustomSchemas(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
+  async importSchemas(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
     const schemas = call.request.params.schemas;
     for (const schema of schemas) {
-      const existingSchema = await this.database
-        .getSchemaModel('_DeclaredSchema')
-        .model.findOne({ name: schema.name });
-      const operation = isNil(existingSchema) ? 'create' : 'update';
-      const imported = schema.modelOptions.conduit.imported === true;
-      const modelOptions = SchemaConverter.getModelOptions({
-        cmsSchema: true,
-        existingModelOptions: schema.modelOptions,
-        importedSchema: imported,
-      });
-      try {
-        validateSchemaInput(schema.name, schema.fields, modelOptions);
-      } catch (err: unknown) {
-        throw new GrpcError(status.INTERNAL, (err as Error).message);
+      const isCmsSchema = schema.modelOptions;
+      if (isCmsSchema) {
+        const existingSchema = await this.database
+          .getSchemaModel('_DeclaredSchema')
+          .model.findOne({ name: schema.name });
+        const operation = isNil(existingSchema) ? 'create' : 'update';
+        const imported = schema.modelOptions.conduit.imported;
+        const modelOptions = SchemaConverter.getModelOptions({
+          cmsSchema: true,
+          existingModelOptions: schema.modelOptions,
+          importedSchema: imported,
+        });
+        try {
+          validateSchemaInput(schema.name, schema.fields, modelOptions);
+        } catch (err: unknown) {
+          throw new GrpcError(status.INTERNAL, (err as Error).message);
+        }
+        await this.schemaController.createSchema(
+          new ConduitSchema(
+            schema.name,
+            schema.fields,
+            modelOptions,
+            schema.collectionName,
+          ),
+          operation,
+          imported,
+        );
       }
-      await this.schemaController.createSchema(
-        new ConduitSchema(
-          schema.name,
-          schema.fields,
-          modelOptions,
-          schema.collectionName,
-        ),
-        operation,
-        imported,
-      );
       if (schema.extensions.length > 0) {
```

Comment:
This doesn't allow for removing extensions.
This endpoint receives a fairly complete representation of the included schemas (save for indexes), therefore missing extensions should be unset.

The only case were dropping extensions would feel wrong is when said extensions are owned by foreign modules. These should most likely be setting their own extensions through gRPC.
As such, I think we should only be setting/unsetting database-owned extensions and either throwing (early) or ignoring any module-specific ones.